otj-metrics-core/src/main/java/com/opentable/metrics/graphite/OtGraphiteReporter.java

```diff
@@ -0,0 +1,207 @@
+package com.opentable.metrics.graphite;
```

Comment:
I'm going to have to take more serious look at thhis. To my mind this has too much copy/paste whhichh makes it impossible to maintain.